content/en/blog/_posts/2022-10-03-introducing-kueue.md

```diff
@@ -0,0 +1,203 @@
+kind: LocalQueue
+metadata:
+  name: training
+  namespace: team-ml
+spec:
+  clusterQueue: research-pool
+```
+
+Admins create the above setup once. Batch users are able to find the queues they are allowed to
+submit to by listing the LocalQueues in their namespace(s). The command is similar to the following: `kubectl get -n my-namespace localqueues`
+
+To submit work, create a Job and set the `kueue.x-k8s.io/queue-name` annotation as follows:
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: sample-job-
+  annotations:
+    kueue.x-k8s.io/queue-name: training
+spec:
+  parallelism: 3
+  completions: 3
+  template:
+    spec:
+      tolerations:
+      - key: spot
+        operator: "Exists"
+        effect: "NoSchedule"
+      containers:
+      - name: example-batch-workload
+        image: registry.example/batch/calculate-pi:3.14
+        args: ["30s"]
+        resources:
+          requests:
+            cpu: 1
+      restartPolicy: Never
+```
+
+
+Kueue intervenes to suspend the Job as soon as it is created. Once the Job is at the head of the ClusterQueue, Kueue evaluates if it can start
+by checking if the resources requested by the job fit the available quota. 
+
+In the above example, the Job tolerates spot resources. If there are previously admitted Jobs consuming all existing on-demand quota but
+not all of spot’s, Kueue admits the Job using the spot quota. Kueue does this by issuing a single update to the Job object that: 
+- Changes the `.spec.suspend` flag to false 
+- Adds the term `instance-type: spot` to the job's `.spec.template.spec.nodeSelector` so that when the pods are created by the job controller, those pods can only schedule 
+  onto spot nodes. 
+
+Finally, if there are available empty nodes with matching node selector terms, then kube-scheduler will directly schedule the pods. If not, then
+kube-scheduler will initially mark the pods as unschedulable, which will trigger the cluster-autoscaler to provision new nodes.
+
+## Future work and getting involved
+The example above offers a glimpse of some of Kueue’s features including support for quota, resource flexibility, and integration with cluster 
+autoscaler. Kueue also supports fair-sharing, job priorities, and different queueing strategies. Take a look at the
+[Kueue documentation](https://github.com/kubernetes-sigs/kueue/tree/main/docs) to learn more about those features and how to use Kueue. 
+
+We have a number of features that we plan to add to Kueue, such as hierarchical quota, budgets, and support for dynamically sized jobs. In 
+the more immediate future, we are focused on adding support for job preemption.
+
+The latest [Kueue release](https://github.com/kubernetes-sigs/kueue/releases) is available on Github;
+try it out if you run batch workloads on Kubernetes.
+We are in the early stages of this project and we are seeking feedback of all levels, major or minor, so please don’t hesitate to reach out. We’re 
```

Comment:
Hi @ahg-g 
Thank you for this nice article as well as the Kueue !

After I read this draft, I visited https://github.com/kubernetes-sigs/kueue and I tried to find what is the required Kubernetes version (1.22 or newer) and the current release version of Kueue (v0.2.1). 

Other readers may want to know this info quickly (like me). So, why don't we directly introduce the required Kubernetes version and the current release version in this article.

I am not an official reviewer for blog. So, please feel free to apply this comment or not.

Reply:
Thanks, I added a note.